### PR TITLE
Documents the cluster's READONLY and READWRITE commands

### PR DESCRIPTION
**Pull request by itamarhaber**

### CODE REVIEW

**Comment by itamarhaber** [comment]
Doesn't look like this PR broke it @djanowski

**Comment by djanowski** [comment]
There's a syntax error in the JSON file (highlighted by GitHub in the diff).

**Comment by itamarhaber** [comment]
Fixes #596

**Comment by charsyam** [comment]
@itamarhaber Thanks. but I have a question.
is it right for summary of read write?
I think it is just turn off read only flags.
but it looks like disable read features.

**Comment by itamarhaber** [comment]
@charsyam `READWRITE` prevents reads from a cluster slave node (the default), so yeah, you could say it kind of disables reads

**Comment by itamarhaber** [comment]
:scream: @badboy is needed in aisle one!

File: commands/client-list.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ b: the client is waiting in a blocking operation
 i: the client is waiting for a VM I/O (deprecated)
 d: a watched keys has been modified - EXEC will fail
 c: connection to be closed after writing entire reply
-u: the client is unblocked
+U: the client is unblocked
```
Comment (badboy) [inline commands/client-list.md]: `u` = `This client was unblocked`

`U` = `Client connected via Unix domain socket`

Comment (itamarhaber) [inline commands/client-list.md]: Redis haz socketz? WOW! :ok_hand:

**Pull request merged**